### DISCUSSION TITLE
Welcome to maitre-d Discussions!

### DISCUSSION THREAD

**Discussion by guillaumeagile** [guillaumeagile/maitre-d#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guillaumeagile** [comment]
Ceux qui veulent collaborer, faites coucou!

**Reply by acassaigne** [reply]
Hello @guillaumeagile ,

Peux-tu m'ajouter au repo ?

Merci

**Reply by superbob** [reply]
Coucou

**Reply by jpboignard** [reply]
yes
